src/cf-local.ts

```diff
@@ -732,20 +732,22 @@ export async function cfGetSpaceServices(
 }
 
 /**
- * * Example of usage : cf bind-local -path .env -service-names serviceName1 serviceName2 -service-keys serviceKeys1 serviceKeys2 -tags tagsValue1 tagsValue2 -params {\"permissions\":[\"development\"]}
+ * * Example of usage : cf bind-local -path .env -service-names serviceName1 serviceName2 -service-keys serviceKeys1 serviceKeys2 -tags tagsValue1 tagsValue2 -params {\"permissions\":[\"development\"]} -quote-vcap
  *
  * @param filePath : string
  * @param instanceNames : string[]
  * @param tags : string[] (Optional)
  * @param serviceKeyNames : string[] (Optional)
  * @param serviceKeyParams : any[] (Optional) Example: {"pemissions":["development"]}
+ * @param quoteVcap : boolean (Optional)
```

Comment:
please consider to add a meaningful description that reflects in docs

Reply:
done